LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
this should already exist

Reply:
It totally does - this was a typo, re-merging your commit into our history.  I can cherrypick it out but it'll require a force-push.